<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-50</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QWidget" name="pageMatriz">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>matriz</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_menu">
     <widget class="QLabel" name="alerta">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>600</width>
        <height>300</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(44, 44, 44);
padding:15px;
border-radius:15px;
font: 700 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>hola wapos</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="fondo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/assets/menu.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="play">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>390</y>
        <width>400</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold;
background-color: rgb(13, 193, 202);
color:#000;
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>PLAY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="crearGrafo">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>600</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold;
background-color: rgb(13, 193, 202);
color:#000;
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>CREAR GRAFO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>181</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>matriz adyacencia</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>491</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DONKEY MOVE DRAFS</string>
      </property>
     </widget>
     <zorder>fondo</zorder>
     <zorder>play</zorder>
     <zorder>crearGrafo</zorder>
     <zorder>alerta</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_7</zorder>
    </widget>
    <widget class="QWidget" name="pageGanaste">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-20</y>
        <width>801</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/assets/ganaste.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>450</y>
        <width>301</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ganaste</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>510</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Jugar otra vez</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_juego">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/assets/background.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pesosColor">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>440</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 174, 255);
color: rgb(0, 0, 0);
padding:15px;
border-radius:15px;
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Pesos&lt;/span&gt;&lt;/p&gt;&lt;p&gt;1&lt;/p&gt;&lt;p&gt;2&lt;/p&gt;&lt;p&gt;3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="pesoNegro">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>500</y>
        <width>63</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pesoNegro_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>530</y>
        <width>63</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(251, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pesoNegro_3">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>560</y>
        <width>63</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagePerdiste">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>211</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Perdiste</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>801</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/assets/perdiste.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>420</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>jugar otra vez</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>label</zorder>
     <zorder>pushButton</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
